bin/export-record-barcodes

```diff
@@ -35,39 +37,141 @@ class HuskyProject(redcap.Project):
         super().__init__(REDCAP_API_URL, REDCAP_API_TOKEN_UW_REOPENING, project_id)
         self.lang = 'en'
         self.purview = 'irb'
+        # There is currently only one arm but with two events for this project:
+        # Enrollment and Encounter events. None of the Enrollment event
+        # isntruments have any barcode fields, so include only the Encounter
+        # event_id.
+        #
+        # -kfay, 30 Sep 2020
+        self.event_id_map = {1: 742155}
 
 
 class ScanProject(redcap.Project):
     lang: str
     purview: Optional[str]
 
-    def __init__(self, project_id: str, lang: str, purview: str = None) -> None:
+    def __init__(self,
+        project_id: int,
+        lang: str,
+        purview: str,
+        event_id_map: Dict[int, int]) -> None:
+        """
+        Creates a SCAN Project object given the REDCap project PID *project_id*,
+        an ISO language code *lang*, a study *purview* (either IRB or Public
+        Health), and a REDCap project *event_id_map* that contains a map of a
+        project's arm number to the associated `event_id` in REDCap.
```

Comment:
Why map arm number to `event_id`? I think the more specific map would be `redcap_event_name` to `event_id`.
This way even if an arm has multiple events, you can map to the correct event.

Reply:
> Why map arm number to event_id?

Because I clearly still don't have a great grasp of REDCap arms vs. events. :sweat_smile: 
I'll try out the `redcap_event_name` to `event_id` mapping.

Reply:
Addressed in a new commit!